File: python/nivel 4 trampas/21.py
"""
Ejercicio 21
Intercambiar dos variables sin usar una tercera
"""
a = int(input("ingrese la variable A: "))
b = int(input("ingrese la variable A: "))

print(f"Antes del intercambio -a -> {a}, b -> {b}")

a, b = b, a

print(f"despues del intercambio  a = {b}, b = {a}")

File: javascript/nivel 4 trampas/21.js
/*
Ejercicio 21
Intercambiar dos variables sin usar una tercera
*/
let a = parseInt(prompt("Ingresa el primer numero: "));
let b = parseInt(prompt("Ingresa el segundo numero: "));

console.log("Antes del Intercambio");
console.log("a = ", a);
console.log("b = ", b);

[a,b] = [b,a];

console.log("Despues del Intercambio");
console.log("a = ", a);
console.log("b = ", b);
